src/PrestaShopBundle/Form/Admin/Type/ConfigurableCountryChoiceType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Type;
+
+use PrestaShop\PrestaShop\Core\Form\ConfigurableFormChoiceProviderInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class responsible for providing configurable countries list
+ */
+class ConfigurableCountryChoiceType extends AbstractType
+{
+    /**
+     * @var ConfigurableFormChoiceProviderInterface
+     */
+    private $countriesChoiceProvider;
+
+    /**
+     * @param ConfigurableFormChoiceProviderInterface $countriesChoiceProvider
+     */
+    public function __construct(ConfigurableFormChoiceProviderInterface $countriesChoiceProvider)
+    {
+        $this->countriesChoiceProvider = $countriesChoiceProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        // Set normalizer enables to use closure for choice generation with options
```

Comment:
Yeah 😅 but ... we also want to keep the close as simple as possible following the [KISS](https://en.wikipedia.org/wiki/KISS_principle) principle. So if we can use a basic `foreach` here 😉 it would be better as we are onboarding the PrestaShop community towards Symfony so let's use it in the most simple way to make the learning curve as smooth as possible

Reply:
example where we have it with foreach would make me do it faster. Searching/thinking about solution.

Reply:
Actually the normalizer gives a nice way to use Options $options values from FormType; Not sure how to do it simpler?!

Reply:
My mistake, I reviewed too fast. Normalizer is used for a good reason here 👍